docs/content/contrib-adding-builtin-functions.md

```diff
@@ -159,3 +160,34 @@ For this example, we add an entry for our new function under the `Strings` secti
 | <span class="opa-keep-it-together">``output := repeat(string, count)``</span> | ``output`` is ``string`` repeated ``count``times | ``SDK-dependent`` |
 ...
 ```
+
+### Add a capability
+
+{{< info >}}
+Read more about extending the default capabilities list for built-ins [here](../deployments/#built-ins).
+{{< /info >}}
+
+One of the security features of OPA is [capabilities](../deployments/#capabilities) checks on policies, allowing users to restrict which built-in functions will be available to policies at runtime.
+To ensure that our new `repeat` function will be available to callers, we'll need to add it to the `capabilities.json` file at the root of the repo.
+
+Since we are adding a new built-in, we will add a new JSON object entry in the list under the `"builtins"` key to describe it:
```

Comment:
I think we should mention that the new entry is auto-generated by `make generate`

Reply:
I did not know that! I'll try that out, and then put it into the docs.

Reply:
Just added some language about `make generate`, and reworded the bit around the JSON chunk to account for it.